distributed/joblib.py

```diff
@@ -116,47 +174,80 @@ def get_nested_backend(self):
     def configure(self, n_jobs=1, parallel=None, **backend_args):
         return self.effective_n_jobs(n_jobs)
 
+    def start_call(self):
+        self.call_data_futures = _WeakKeyDictionary()
+
+    def stop_call(self):
+        # The explicit call to clear is required to break a cycling reference
+        # to the futures.
+        self.call_data_futures.clear()
+        self.call_data_futures = None
```

Comment:
Why set this to `None`? If `call_data_futures` always exists and is just cleared after running, then you never need to check if it's `None` below.

Reply:
Alright.